src/monitors/autoreact.ts

```diff
@@ -18,6 +18,8 @@ botCache.monitors.set("autoreact", {
     const settings = await db.autoreact.get(message.channelID);
     if (!settings) return;
 
+    if (message.author.bot && !botCache.vipGuildIDs.has(message.guildID)) return;
```

Comment:
This won't work.

- The `ignoreBots` option on the monitor needs to be set to false so this monitor runs on bots.
- Not all vip servers should have this only select ones that need it. I can manually add some server ids for it. Just a const SPECIAL_SERVER_IDS array new Set at the top will work.
- if (message.author.bot && !SPECIAL_SERVERS.has(message.guildID)) return

Reply:
Also move this before the db fetch to save a db call